### DISCUSSION TITLE
Student Pack

### DISCUSSION THREAD

**Discussion by abdullahbaris21** [community/community#37494] [GitHub Education]
I'm trying to fill out a document but I get the error "File is too small"

**Comment by DSDmark** [comment]
try to upload another file

**Reply by abdullahbaris21** [reply]
> try to upload another file

i did

**Reply by DSDmark** [reply]
How much **KB** your file have?

**Reply by DSDmark** [reply]
Hey @abdullahbaris21,

I'm not sure if it could help you, but You could try this :- 

Go there [**image size increase**](https://www.imgonline.com.ua/eng/resize-image.php)

Select width and height 1920x1080 and in the bottom there is an **Output image format** section select **PNG-24 (without compression, with transparency support)**  click on **ok** Button your size will be increase.

**Reply by abdullahbaris21** [reply]
thanks working
